R/pkg/DESCRIPTION

```diff
@@ -18,10 +18,10 @@ Collate:
     'schema.R'
     'generics.R'
     'jobj.R'
-    'RDD.R'
-    'pairRDD.R'
     'column.R'
     'group.R'
+    'RDD.R'
```

Comment:
Yeah some versions of `roxygen` might interpret the order differently or something like that. Its best to leave this file untouched by this PR and we can do a separate PR to bring `DESCRIPTION` up to date.